distribution/pom.xml

```diff
@@ -43,6 +43,7 @@
     <properties>
         <!-- the default value is a repeated flag from the command line, since blank value is not allowed -->
         <druid.distribution.pulldeps.opts>--clean</druid.distribution.pulldeps.opts>
+        <jdk.version>11</jdk.version>
```

Comment:
I would make this property name more explicit to avoid confusion with the `java.version` property already defined for the maven build.

Reply:
How about `docker.jdk.version`?